### PR TITLE
ESLint config, new panel & new button variants

### PR DESCRIPTION
**Pull request by rdjpalmer**
1. ESLint config now matches topnotes, using Airbnb's configuration
2. Panel component has a "squared" variant
3. Button component has a "clean" variant

### CODE REVIEW

File: components/Btn/Btn.css [hunk 1]
```diff
@@ -126,6 +126,30 @@
...
+.primary.clean {
+  color: var(--color-primary);
+}
+.primary.clean:hover {
+  color: color(var(--color-primary) l(- 10%));
+}
+.primary.clean:active {
+  color: color(var(--color-primary) a(70%));
+}
+
```
Comment (iest) [inline components/Btn/Btn.css]: :ok_hand:

File: styleguide/sections/Buttons.js [hunk 2]
```diff
@@ -16,16 +16,24 @@ export default class BtnSection extends Component {
...
-        <p>With icon(s):</p>
+        <h4>Clean</h4>
+
+        <Btn className={[m.mbs, m.mrs].join(' ')} variant="clean">Clean</Btn>
+        <Btn className={[m.mbs, m.mrs].join(' ')} type="primary" variant="clean">Primary</Btn>
+
+        <h3>With icon(s)</h3>
 
         <Btn className={m.mbs}>
           <Icon name="heart"/>
```
Comment (iest) [inline styleguide/sections/Buttons.js]: D'you wanna add an icon one for each variant?

File: styleguide/sections/Colours.js [hunk 3]
```diff
@@ -59,7 +59,7 @@ export default class Colours extends Component {
           'error',
           'alert',
           'text',
-          'muted'].map((c, i) => <Swatch key={i} colour={c}/>)
+          'muted', ].map((c, i) => <Swatch key={i} colour={c}/>)
```
Comment (iest) [inline styleguide/sections/Colours.js]: This would look less horrible if the array bits were on a newline. But ¯\_(ツ)_/¯

**Comment by iest** [comment]
eelygood

**Comment by iest** [comment]
bender-pimp

**Comment by iest** [comment]
What's that merge doing there?

**Comment by rdjpalmer** [comment]
Much confusion. Gitup is complicating things for me :)

**Comment by iest** [comment]
:ship:

**Pull request merged**